crates/sui-source-validation-service/src/lib.rs

```diff
@@ -37,7 +52,26 @@ pub async fn verify_package(
         .map_err(anyhow::Error::from)
 }
 
-pub async fn initialize(
+pub fn parse_config(config_path: impl AsRef<Path>) -> anyhow::Result<Config> {
+    let contents = fs::read_to_string(config_path)?;
+    toml::from_str(&contents).map_err(anyhow::Error::from)
+}
+
+pub async fn clone_repositories(config: &Config) -> anyhow::Result<()> {
+    for p in &config.packages {
+        let _ = p.repository;
+        let _ = p.paths;
+    }
+    Ok(())
+}
```

Comment:
Added to remove `unused` warning, will be implemented in following PR

Reply:
You could probably just prefix `config` with an underscore, i.e. `_config`.